getsong/getsong.py

```diff
@@ -37,18 +38,14 @@ def main():
     if not uri:
         print("Could not find result for {}. http://youtube.com/results?{}".format(term, urlencode({'search_query': term})))
         return 2
+    response = input("About to download the audio for {}.\nIs this correct? [y/n] ".format(title))
     try:
-        input = raw_input
```

Comment:
This caused an error for me in Python 3, so I just used `from builtins import input`